EKF/mag_control.cpp

```diff
@@ -70,7 +70,8 @@ void Ekf::controlMagFusion()
 		return;
 	}
 
-	if (canRunMagFusion()) {
+	if (shouldRunMagFusion()) {
+		_mag_yaw_reset_req |= !_control_status.flags.mag_hdg;
```

Comment:
I guess this is changing the change indicator output. I am a bit surprised that this change is also affecting the output for a model that is not using any vision measurement.

Reply:
This will continuously request resets in `mag_3D` mode.